spec/models/sitemap_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+describe Sitemap do
+  let(:url) { 'http://agency.gov/sitemap.xml' }
+  let(:valid_attributes) { { url: url } }
+
+  it { is_expected.to have_readonly_attribute(:url) }
+
+  describe 'schema' do
+    it { is_expected.to have_db_column(:url).of_type(:string).with_options(null: false, limit: 2000) }
+  end
+
+  describe 'validations' do
+    context 'when the sitemap\'s domain is invalid' do
```

Comment:
No need to escape the apostrophe -- you can simply use double quotes:
```ruby
context "when the sitemap's domain is invalid" do
```
I'll clean this up, and move this spec to the `shared_examples_for 'a record that belongs to a searchgov_domain'` section.